### DISCUSSION TITLE
Parent component click events block child component click events

### DISCUSSION THREAD

**Discussion by zhengtulymGh** [Budibase/budibase#9756] [Help]
I set a click action on the parent container, and I can't select the picker in the form, which means only the click event of the parent component was triggered.
![image](https://user-images.githubusercontent.com/13768025/220284990-2dde0d51-ebda-4832-bb39-38059d349242.png)

![image](https://user-images.githubusercontent.com/13768025/220285308-968ebfec-a8d1-4eff-9051-91cef20290d9.png)

How to trigger the click events of both the child component and parent component?

**Comment by melohagan** [comment]
Hey @zhengtulymGh 

If you want to access a binding then the reference has to be in a child component.

I.e. 
> Parent
> > Child (can access Parent values through bindings)

So in your case you'd need your tree to look roughly like follows:

> Form
> > Container (can bind to any of the field values of the form)
> > > Breadcrumbs Container
> > > Title Container
> > > Field Group

**Reply by zhengtulymGh** [reply]
Maybe I didn't explain it clearly. What I meant was the bubbling issue of the click event, rather than accessing a binding of the parent component.

**Reply by melohagan** [reply]
Ah yes I understand. This is done on purpose. If you add a click event to a container then it will disable click events for child components to avoid multiple sets of actions being triggered by one click.

**Reply by zhengtulymGh** [reply]
But sometimes I need to add a click event to the top-level container. For example, I added a pop-up prompt in the header and I want to hide this prompt by clicking on other areas of the page.

**Reply by melohagan** [reply]
Hey @zhengtulymGh Is that the **Show Notification** action you are using? You could tick the *Auto dismiss* checkbox.

**Reply by zhengtulymGh** [reply]
No, it's a dialog box I configured myself.
